curl-helper.c

```diff
@@ -641,14 +643,16 @@ static void resetOcamlValues(Connection* connection)
     int i;
 
     for (i = 0; i < OcamlValuesSize; i++)
-        Store_field(connection->ocamlValues, i, Val_unit);
+      if (i != Ocaml_CALLBACKBUFFER)
```

Comment:
this wouldn't be needed if bigarray is allocated only when used

Reply:
Agreed, done.